test: Skip bison tests if bison is not available

Explorer09 opened the pull request:
This pull request depends on #144 
Resolves #49, except that this configure script is not tolerant on other yacc implementations (byacc, for example). If someone wants byacc for building flex and/or the testsuite, just tweak the code then.

westes commented:
Would you put the README change on a separate pr, please. It is not related to the bison changes.

Explorer09 commented:
@westes 
The README change depends on the previous commit ("Skip bison tests" one). I'm not sure if it's a good idea to make it in a separate PR, then. If I were to separate it, I would rather have `testwrapper.sh` updated to be aware of "skipped" exit status as well.

Explorer09 commented:
Update: the tests/README change is on separate pull request now: #148

westes commented:
There's conflicts now. Would you fix that and push the update? Go ahead and squash/fixup the history as appropriate, too.

westes commented:
You need to add the no_bison_stub.c.

Explorer09 commented:
Rebased and added no_bison_stub.c. (And would you please be patient and wait a bit before I say it's okay? I was tweaking the no_bison_stub.c when I discovered that it's not included in the commit just now.)

westes commented:
You bet, just holler when.

On Tuesday, 24 January 2017,  5:34 pm -0800, "Kang-Che Sung (宋岡哲)" <notifications@github.com> wrote:

> Rebased and added no_bison_stub.c. (And would you please be patient and wait a bit before I say it's okay? I was tweaking the no_bison_stub.c when I discovered that it's not included in the commit just now.)
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/westes/flex/pull/145#issuecomment-274994230

-- 
Will Estes
westes575@gmail.com

Explorer09 commented:
It's ready to pull now.

westes commented:
Thanks. This is on master and will be included in the next release.
